bamboo/README.md

```diff
@@ -18,17 +18,28 @@ Each job has the same basic structure:
 
 # Directory Structure
 
-'bamboo/compiler_tests', 'bamboo/integration_tests', 'bamboo/unit_tests' each have a 'conftest.py' that pytest requires. They also contain one or more python files. Each file has a number of tests to run. pytest recognizes methods that begin with `test_` as tests. Test methods should use the `assert` keyword.
+'bamboo/compiler_tests', 'bamboo/integration_tests', 'bamboo/unit_tests' each have a 'conftest.py' that pytest requires. They also contain one or more python files. Each file has a number of tests to run. 
+
+# Writing Your Own Tests
+
+The way we have Bamboo setup requires having tests use pytest. pytest recognizes methods that begin with `test_` as tests. Test methods should use the `assert` keyword.
```

Comment:
Worded a bit awkwardly. "Tests must be written using the python pytest module."

Reply:
Hm, ""Tests must be written using the python pytest module" is meaningful only if you're familiar with pytest. IMO, the current wording is more informative. Though I think the last statement needs amplyifying. I think it means that if you write a test (a method that begins with test_) it will fail if an assert() within the test fails. Otherwise it passes -- or doesn't it also fail if it returns non-zero?